plugin-modernizer-core/src/main/resources/urls.properties

```diff
@@ -1,3 +1,4 @@
 update.center.url=https://updates.jenkins.io/current/update-center.actual.json
 plugin.versions.url=https://updates.jenkins.io/current/plugin-versions.json
 plugin.health.score.url=https://plugin-health.jenkins.io/api/scores
+plugin.stats.top.url=https://stats.jenkins.io/jenkins-stats/svg/202406-plugins.csv
```

Comment:
I don't think we need some logic to change the month. We don't really care if we have very recent results. The idea is probably to focus on most used plugin for refactoring tasks